parser/parser.go

```diff
@@ -9,6 +9,8 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/open-policy-agent/conftest/parser/spdx"
+
```

Comment:
nit: Unnecessary newline and imports should be in alphabetical order. `gofmt` should fix this automatically,

Reply:
thank you so much @jalseth, done.